shared/api/endpoints/createFeedbackEndpoint.ts

```diff
@@ -85,8 +85,8 @@ const getFeedbackType = (routeType: FeedbackRouteType, slug?: string): FeedbackT
   }
 }
 
-export default (baseUrl: string): Endpoint<ParamsType, void> =>
-  new EndpointBuilder<ParamsType, void>(FEEDBACK_ENDPOINT_NAME)
+export default (baseUrl: string): Endpoint<ParamsType, Record<string, never>> =>
+  new EndpointBuilder<ParamsType, Record<string, never>>(FEEDBACK_ENDPOINT_NAME)
```

Comment:
❓ I don't quite understand the never part here. Is that supposed to be overwritten when calling the EndpointBuilder?

Reply:
Ooooh, cool, thanks for the explanation!